src/Transition/CreateNewIDFUsingRulesV9_2_0.f90

```diff
@@ -486,6 +497,369 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
+                  POutArgs(7) = InArgs(23 + iPt)  ! Unit type
+
+                  POutArgs(6) = Blank
+                  POutArgs(8:10) = Blank
+
+                  RefIndex = 31 + NumIndVars
+                  DO iPt2=1,iPt-1
+                    RefIndex = RefIndex + NumIndVarsVals(iPt2)
+                  END DO
+
+                  IndVars(iPt,1:NumIndVarsVals(iPt)) = InArgs(RefIndex + 1:RefIndex + NumIndVarsVals(iPt))
+
+                  POutArgs(11:PArgs) = IndVars(iPt,1:NumIndVarsVals(iPt))
+
+                  CALL WriteOutIDFLines(DifLfn,'Table:IndependentVariable',PArgs,POutArgs,PFldNames,PFldUnits)
+                END DO
+
+
+
+                ! Create new Table:IndependentVariableList object
+                PArgs = 1 + NumIndVars
+                CALL GetNewObjectDefInIDD('TABLE:INDEPENDENTVARIABLELIST',PNumArgs,PAorN,NwReqFld,PObjMinFlds,PFldNames,PFldDefaults,PFldUnits)
+                POutArgs(1) = TRIM(InArgs(1))//'_IndependentVariableList'
+
+                DO iPt=1,NumIndVars
+                  POutArgs(1 + iPt) = TRIM(InArgs(1))//'_IndependentVariable'//RoundSigDigits(iPt)
+                END DO
+                CALL WriteOutIDFLines(DifLfn,'Table:IndependentVariableList',PArgs,POutArgs,PFldNames,PFldUnits)
+
+
+
+                ! Create new Table:Lookup object
+                CALL GetNewObjectDefInIDD('TABLE:LOOKUP',PNumArgs,PAorN,NwReqFld,PObjMinFlds,PFldNames,PFldDefaults,PFldUnits)
+                POutArgs(1) = TRIM(InArgs(1))
+                POutArgs(2) = TRIM(InArgs(1))//'_IndependentVariableList'
+
+                ! Normalization Value
+                IF (InArgs(13) == Blank) THEN
+                  POutArgs(3) = Blank
+                  POutArgs(4) = Blank
+                ELSE
+                  POutArgs(3) = 'Yes'
+                  POutArgs(4) = InArgs(9)  ! Normalization Value
+                END IF
+                POutArgs(5:6) = InArgs(22:23) ! Min/Max
+                POutArgs(7) = InArgs(30)  ! Units
+                POutArgs(8:10) = Blank  ! External file
+
+                IF (InArgs(6) /= Blank) THEN
+                  CALL writePreprocessorObject(DifLfn,PrognameConversion,'Warning',  &
+                          'Table:MultivariableLookup="'//trim(InArgs(1))//  &
+                                  '" references an external file="'//  &
+                                  trim(InArgs(6))//'". External files must be converted to the new format using'//&
+                                  'table_convert.py.')
+                  write(diflfn,fmtA) ' '
+                  CALL ShowWarningError('Table:MultivariableLookup="'//trim(InArgs(1))//  &
+                          '" references an external file="'//  &
+                          trim(InArgs(6))//'". External files must be converted to the new format using'//&
+                          'table_convert.py or by appending the contents of the external file to the original IDF object.',Auditf)
+                END IF
```

Comment:
OK, so there is a path forward for those external file users who don't run Python, they can just stick the contents back in their IDF and re-run and it will show up fine.